src/control/server/instance_storage_rpc.go

```diff
@@ -28,9 +28,14 @@ import (
 )
 
 var (
-	scanSmd                       = listSmdDevices
-	getCtrlrHealth                = getBioHealth
+	// Function pointers to enable mocking.
+	scanSmd         = listSmdDevices
+	getCtrlrsHealth = getBioHealth
```

Comment:
This is very confusing now that there is a regular function named `getCtrlrHealth`

Reply:
I thought it wasn't too bad given one is plural, getCtrlrsHealth and getCtrlrHealth?

Reply:
This makes it easier to tell the difference at a glance, thanks.